packages/botkit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "botkit",
-  "version": "4.8.1",
+  "version": "4.8.2",
```

Comment:
i'll handle this don't worry about changing version number

Reply:
I reverted the version.